doc/advanced_setup.rst

```diff
@@ -103,6 +103,29 @@ runtime accordingly.
 If you use another Python setup and you encounter some difficulties please
 report them on the MNE mailing list or on github to get assistance.
 
+Installing Mayavi
+^^^^^^^^^^^^^^^^^
+
+If you have Anaconda installed (recommended), the easiest way to install `mayavi` is to do::
+
+    conda install mayavi
```

Comment:
(even if we omit `>>>` for Python code, we should at least put the `$` here so people have more indication that it's not Python)